webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderUserDetails.scala

```diff
@@ -37,21 +37,15 @@
 
 package bootstrap.liftweb
 
-import java.io.File
-import java.io.FileInputStream
-import java.io.InputStream
+import java.io.{File, FileInputStream, InputStream}
```

Comment:
Soory, I mis-explained: it's the opposite that is needed: one import by line, ie we want: 

```
import java.io.File
import java.io.FileInputStream
import java.io.InputStream
```
Not: 
```
import java.io.{File, FileInputStream, InputStream}
```